src/datatype/text_bvecf32.rs

```diff
@@ -13,28 +13,27 @@ fn _vectors_bvecf32_in(input: &CStr, _oid: Oid, typmod: i32) -> BVecf32Output {
         .dims()
         .map(|x| x.get())
         .unwrap_or(0);
-    let v = parse_vector(input.to_bytes(), reserve as usize, |s| {
-        s.parse::<u8>().ok().and_then(|x| match x {
-            0 => Some(false),
-            1 => Some(true),
-            _ => None,
-        })
-    });
-    match v {
-        Err(e) => {
-            bad_literal(&e.to_string());
+    let mut bool_vec = Vec::<bool>::with_capacity(reserve as usize);
```

Comment:
Why?

Reply:
I want to use `parse_vector` for `svector`. Therefore, I changed `parse_vector` to pass all the strs that need to be parsed to a hook function instead of returning a vector.

Reply:
We can just let `parse_vector` for `svector` return `Vec<F32>` too. Allocation is not a big ideal.

Reply:
update `parse_vector` to previous impl in new commit